### ISSUE TITLE
Being able to put in custom sound files.

### ISSUE THREAD

**Issue by Foxxel** [xchataqua/xchataqua#173]
Trying to figure out a way to be able to put custom sound files into the program. Need/want to put a EAS tone for NOAA IRC channel to alert. Also if there is a way to make it so you can use different alerts on different servers that would be awsome.

**Comment by challinan** [comment]
I crafted a total hack for this, because I wanted a unique sound for each channel.  You can easily add this.  I am traveling and don't have access to my build machine, but when I return I will post a commit to my cloned version of this tree, or you can contact me directly.

**Comment by Foxxel** [comment]
Yeah totally msg me back when you can, I'd totally be down to figuring out how to do that. Would be super helpful~!

**Comment by youknowone** [comment]
@challinan it sounds cool ;) thanks for sharing it
